business-registry-dashboard/app/components/Modal/RemoveBusiness/Generic.vue

```diff
@@ -9,14 +9,21 @@ defineEmits<{
   confirm: [void]
   close: [void]
 }>()
+function selectDescription (removeBusinessPayload: RemoveBusinessPayload) {
+  // Different discriptions for Continuation In's if they have a name request
+  if (removeBusinessPayload.business.corpType.code === CorpTypes.CONTINUATION_IN && removeBusinessPayload.business.nameRequest) {
```

Comment:
What about the other 3 continuation in types (CBEN, CCC, CUL)?

Reply:
Oh, never mind. This is one of those overloaded values.

![image](https://github.com/user-attachments/assets/5f611f80-9846-4c1b-ba52-6b2a7e56885e)

Resolving.